core/msd/src/main/rdl/KubernetesNetworkPolicy.tdl

```diff
@@ -0,0 +1,74 @@
+// Copyright The Athenz Authors
+// Licensed under the terms of the Apache version 2.0 license. See LICENSE file for terms.
+
+include "Names.tdl";
+include "TransportPolicyRule.tdl";
+
+// A label selector requirement is a selector that contains values, a key, and an operator that relates the key and values.
+type KubernetesLabelSelectorRequirement Struct {
+    String key; //Label key that the selector applies to
+    String operator; //Operator that is applied to the key. Valid operators are In, NotIn, Exists and DoesNotExist.
+    Array<String> values (optional); //Array of string values. If the operator is In or NotIn, the values array must be non-empty. If the operator is Exists or DoesNotExist, the values array must be empty.
+}
+
+// A label selector is a label query over a set of resources. The result of matchLabels and matchExpressions are ANDed. An empty label selector matches all objects. A null label selector matches no objects.
+type KubernetesLabelSelector Struct {
+    Array<KubernetesLabelSelectorRequirement> matchExpressions; //Array of label selector requirements. The requirements are ANDed.
+    Map<String, String> matchLabels; //Map of label key/value pairs
```

Comment:
`matchExpressions` can accommodate this requirement as well, with a new `isEqual` operator or existing `In`